service/history/handler.go

```diff
@@ -1030,6 +1034,7 @@ func (h *handlerImpl) SignalWorkflowExecution(
 	defer log.CapturePanic(h.GetLogger(), &retError)
 	h.startWG.Wait()
 
+	ctx = withSignalName(ctx, wrappedRequest.SignalRequest.GetSignalName())
```

Comment:
As this is for customer use, we just need to add it in front end service. No need to add the tag in history.